core/server/api/utils.js

```diff
@@ -245,13 +246,13 @@ utils = {
         };
     },
 
-    trimAndLowerCase: function trimAndLowerCase(params) {
+    trimAndLowerCase: (params) => {
         params = params || '';
         if (_.isString(params)) {
             params = params.split(',');
         }
 
-        return _.map(params, function (item) {
+        return params.map((item) => {
             return item.trim().toLowerCase();
         });
     },
```

Comment:
As a general rule, we are converting all same-name functions inside an object to arrow functions. Exception to these rules are functions - `prepareInclude`, `prepareFields` and `prepareFormats` - in this object, as they use another function in this object with `this. trimAndLowerCase`. This is where converting named functions in an object to arrow does not work, as it loses the scope of `this`. Not sure if its best to keep them as exceptions or just not convert object named functions to arrow functions